docs/yaml-monitoring-templates.rst

```diff
@@ -177,7 +177,7 @@ Monitoring Template Fields
 The following fields are valid for a monitoring template entry.
 
 name
-  :Description: Name (e.g. WidgeterHealth). Must be a valid Zenoss object ID.
+  :Description: Name (e.g. WidgeterHealth). Must be a valid Zenoss object ID.  ZenPackLib also allows for defining a replacement or additional template by adding "-replacement" or "-additional" to the end of the template name.  For example, a defined *Device-replacement* template will replace the existing Device template on a device class.  A defined *Device-addition* template will be applied in addition to the existing Device template on a device class.
```

Comment:
Dave, would you mind splitting this out into a "note" field like the ones for "extra_paths" and "order"?